engine/app/controllers/good_job/base_controller.rb

```diff
@@ -30,8 +30,8 @@ def default_url_options(options = {})
 
     private
 
-    def set_locale
-      I18n.locale = current_locale
+    def switch_locale(&action)
+      I18n.with_locale(current_locale, &action)
```

Comment:
what do you prefer `with_locale`?

Reply:
Good question. I like that `with_locale` only changes the locale within the context of the block and then returns to the previous value. Because the Goodjob engine is running in the context of a parent application, I don't want to change the `I18n.locale` value globally because it might have an unexpected side effect in the parent application.

Reply:
Good to know, yes, we don't want unexpected behavior 👍

Thank you for the info